packages/gestalt/src/Checkbox.js

```diff
@@ -69,6 +71,14 @@ export default class Checkbox extends React.Component<Props, State> {
     this.props.onChange({ event, checked });
   };
 
+  handleClick = (event: SyntheticInputEvent<>) => {
```

Comment:
`event: SyntheticInputEvent<HTMLInputElement>` so we can properly flow type the event target